langserver/hover.go

```diff
@@ -47,6 +48,26 @@ func initializeFunctionDocumentation() http.FileSystem {
 	return ret
 }
 
+func compatibleVersion(originalVersion, requiredVersion string) bool {
```

Comment:
In case of `s.headless` being set, i.e. the language server being used through the `rest` package, `connectPrometheus` is never called. However in that case it makes sense to always assume the latest compatible version, since currently the only planned user for the `rest` package is Prometheus itself.

Reply:
We can set the latest Prometheus version when the rest package is called directly?

Reply:
That will solve the issue.

Reply:
That would make it necessary to update the language server every time there is a new Prometheus release and increase maintenance overhead.

Reply:
:confused: Any ideas

Reply:
See suggestion.